core/src/main/java/com/datastax/oss/driver/api/core/cql/Statement.java

```diff
@@ -18,6 +18,7 @@
 import com.datastax.oss.driver.api.core.CqlIdentifier;
 import com.datastax.oss.driver.api.core.CqlSession;
 import com.datastax.oss.driver.api.core.config.DriverConfigProfile;
+import com.datastax.oss.driver.api.core.context.DriverContext;
```

Comment:
The JAVA-708 PR (#978) also introduced new metrics for measuring the number of bytes/received sent (via Inbound/OutboundTrafficMeter channel handlers),  we should make sure to add metrics for that, but it doesn't need to be done in this PR.  I created [JAVA-1829](https://datastax-oss.atlassian.net/browse/JAVA-1829) for that.